test/specs/repo/extensions/nuke.js

```diff
@@ -18,7 +18,7 @@ describe( 'Nuke', function () {
 			'Vandals In Motion'
 		);
 
-		const result = await browser.makeRequest(
+		let result = await browser.makeRequest(
```

Comment:
Can you make `result` two different variables, ideally with some indication of what each is the result of?

Reply:
readability first. fully agree

Reply:
done